src/webviews/extension-side/plotView/helpers.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { NotebookCellOutput } from 'vscode';
+import { CellOutputMetadata } from '../../../kernels/execution/helpers';
+
+export function getCellOutputMetadata(cell?: NotebookCellOutput): CellOutputMetadata | undefined {
```

Comment:
We probably don't need a helper function for this, `cell?.metadata as CellOutputMetadata | undefined`